utils/distarray_conda-env.yml

```diff
@@ -0,0 +1,44 @@
+name: distarray
+dependencies:
```

Comment:
This file is actually auto generated by conda, so I assume that can be done. I will run some tests and update this.